shared/data/surrogates.txt

```diff
@@ -0,0 +1,360 @@
+google-analytics.com/ga.js application/javascript
```

Comment:
I'm not a fan of this format - these should be separate JS files IMO. Is there any good reason to keep it? (thats something for org days, not for this PR)

Reply:
I don't like it either, but I'd change the format later.

Reply:
Agreed, this format is bad and these should be in individual files. Let's plan to fix it as a follow up when we check everything into Tracker Radar?